pkg/volume/csi/csi_client.go

```diff
@@ -639,7 +639,7 @@ func (c *csiDriverClient) NodeGetVolumeStats(ctx context.Context, volID string,
 			metrics.Inodes = resource.NewQuantity(usage.GetTotal(), resource.BinarySI)
 			metrics.InodesUsed = resource.NewQuantity(usage.GetUsed(), resource.BinarySI)
 		default:
-			klog.Errorf("unknown key %s in usage", unit.String())
+			klog.ErrorS(nil, "unknown key in usage", "unitKey", unit.String())
```

Comment:
The original error here was a little unclear, maybe better to do "unknown unit in VolumeUsage", "unit", unit.String()?

Reply:
Your suggestion is better, I will update it.